tools/swift-api-digester/ModuleAnalyzerNodes.cpp

```diff
@@ -1368,11 +1368,26 @@ SwiftDeclCollector::constructExternalExtensionNode(NominalTypeDecl *NTD,
                                             ArrayRef<ExtensionDecl*> AllExts) {
   auto *TypeNode = SDKNodeInitInfo(Ctx, NTD).createSDKNode(SDKNodeKind::DeclType);
   addConformancesToTypeDecl(cast<SDKNodeDeclType>(TypeNode), NTD);
+
+  bool anyConformancesAdded = false;
   // The members of the extensions are the only members of this synthesized type.
   for (auto *Ext: AllExts) {
     HandledExtensions.insert(Ext);
     addMembersToRoot(TypeNode, Ext);
+
+    // Keep track if we've declared any conformances in this extension.
+    // FIXME: This will also add conformances to internal protocols, so this
+    //        should be augmented to handle internal protocols or conditional
+    //        conformances.
```

Comment:
May be worth mentioning the tricky case of "InternalProto : PublicProto", so that whoever tries to fix it doesn't do so naïvely.